single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/model/nodes/feeders/FeederTwLeg.java

```diff
@@ -15,8 +15,11 @@
  */
 public class FeederTwLeg extends FeederWithSides {
 
+    VoltageLevelInfos ownVoltageLevelInfos;
```

Comment:
```suggestion
    private final VoltageLevelInfos ownVoltageLevelInfos;
```
Parameters should usually be private and accessed through getters. When they don't need to change they should also be marked final

Reply:
I forgot to delete this attribute actually. I do not use it at the moment.